browser-extension/mv3/src/page-scripts/ajaxRequestInterceptor/xhr.js

```diff
@@ -121,7 +121,7 @@ export const initXhrInterceptor = (debug) => {
         }
 
         if (typeof customResponse === "undefined") {
-          return;
+          customResponse = this.response;
```

Comment:
I don't think this is the only thing that will solve this issue. if `getFunctionFromCode` throws an error, this if won't event capture it